.eslintignore

```diff
@@ -6,6 +6,7 @@ public
 .next
 .*cache
 *.min.js
+!/.*.js
```

Comment:
is this intended to ignore many files?
Or just `.remarkrc.js`?

Reply:
.*.js files are ignored by ESLint by default, `!` makes they are not ignored.